src/components/Modal/Terms.vue

```diff
@@ -34,18 +34,15 @@
 <script>
 import { getUrl } from '@snapshot-labs/snapshot.js/src/utils.ts';
 export default {
-  props: ['open', 'space'],
+  props: { open: Boolean, space: Object },
   emits: ['close'],
-  methods: {
-    accept() {
-      this.$emit('accept');
-      this.$emit('close');
-    }
-  },
-  computed: {
-    getIpfsUrl() {
-      return getUrl(this.space.terms);
+  setup(props, { emit }) {
+    function accept() {
+      emit('accept');
+      emit('close');
     }
+
+    return { accept, getIpfsUrl: getUrl(props.space.terms) };
```

Comment:
In a proposal when space doesn't have a terms field, getting this error (ex: yam space / gitcoingdao space)
![image](https://user-images.githubusercontent.com/15967809/124122505-c9770400-da93-11eb-8cab-45281aeadd0a.png)

Reply:
Ah nice spot